tests/validation/tests/v3_api/test_bkp_restore_local.py

```diff
@@ -22,6 +24,131 @@ def test_bkp_restore_local_delete():
     validate_backup_delete(ns, binfo)
 
 
+@if_test_rbac
+def test_rbac_create_bkp_restore_local_cluster_owner():
+    """ Only cluster-owner should be allowed to create backups """
+    user_token = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    user_client = get_client_for_token(user_token)
+    user_cluster = user_client.list_cluster(name=CLUSTER_NAME).data[0]
+    backup = user_cluster.backupEtcd()
+    backupname = backup['metadata']['name']
+    wait_for_backup_to_active(user_cluster, backupname)
+    assert len(user_cluster.etcdBackups(name=backupname)) == 1
+
+
+@if_test_rbac
+@pytest.mark.parametrize("role", rbac_roles)
+def test_rbac_create_bkp_restore_local_cluster_unpriv(role):
+    """
+    Only cluster-owner should be allowed to create backups
+    unprivileged user should get 403 PermissionDenied
+    """
+    user_token = rbac_get_user_token_by_role(role)
+    user_client = get_client_for_token(user_token)
+    user_cluster = user_client.list_cluster(name=CLUSTER_NAME).data[0]
+    with pytest.raises(ApiError) as e:
+        user_cluster.backupEtcd()
+    assert e.value.error.status == 403
+    assert e.value.error.code == 'PermissionDenied'
+
+@if_test_rbac
+def test_rbac_list_bkp_restore_local_cluster_owner():
+    """ Only cluster-owner should be allowed to list backups """
+    user_token = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    user_client = get_client_for_token(user_token)
+    user_cluster = user_client.list_cluster(name=CLUSTER_NAME).data[0]
+    backup = user_cluster.backupEtcd()
+    backupname = backup['metadata']['name']
+    assert len(user_cluster.etcdBackups(name=backupname)) == 1
+
+
+@if_test_rbac
+@pytest.mark.parametrize("role", rbac_roles)
+def test_rbac_list_bkp_restore_local_cluster_unpriv(role):
+    """
+    unprivileged user should not be allowed to list backups
+    cluster etcdBackups() should always return length zero
+    """
+    user_token = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    user_client = get_client_for_token(user_token)
+    user_cluster = user_client.list_cluster(name=CLUSTER_NAME).data[0]
+    backup = user_cluster.backupEtcd()
+    backupname = backup['metadata']['name']
+    assert len(user_cluster.etcdBackups(name=backupname)) == 1
+    wait_for_backup_to_active(user_cluster, backupname)
+    user_token2 = rbac_get_user_token_by_role(role)
+    user_client = get_client_for_token(user_token2)
+    user_cluster2 = user_client.list_cluster(name=CLUSTER_NAME).data[0]
+    assert len(user_cluster2.etcdBackups()) == 0
+
+
+@if_test_rbac
+@pytest.mark.parametrize("role", rbac_roles)
+def test_rbac_restore_bkp_restore_local_cluster_unpriv(role):
```

Comment:
Can we maybe change the method names, they seem redundant and long? 
`test_rbac_restore_bkp_restore_local_cluster_unpriv`  to `test_rbac_restore_local_1`

`test_rbac_list_bkp_restore_local_cluster_unpriv` to `test_rbac_list_local_1` (1 if more than 1 similar tests are there)

Reply:
I've refactored the methods names.